cbr-service-profile/main.tf

```diff
@@ -28,6 +28,7 @@ locals {
 
   vpc_zone_list = (length(var.zone_vpc_crn_list) > 0) ? [{
     name             = "${var.prefix}-cbr-vpc-zone"
+    region           = var.region_vpc_zone_list
```

Comment:
It's likely not needed for vpcs given that a vpc is already in a single region only. Region is only needed for the service list.

Reply:
Updated